src/containers/Tag/Tag.tsx

```diff
@@ -93,12 +93,12 @@ export const Tag: React.SFC<TagProps> = (props) => {
 
   const languageOptions = languages.data
     ? languages.data.languages.map((language: any) => {
-      return (
-        <MenuItem value={language.id} key={language.id}>
-          {language.label}
-        </MenuItem>
-      );
-    })
+        return (
```

Comment:
Not sure why formatting is picked here. Can you please run "yarn format". Make sure to update your branch.